<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dash</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../assets/escavadeira.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gestor de Locações</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Equipamentos Pesados</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNovaLocacao">
     <property name="geometry">
      <rect>
       <x>794</x>
       <y>22</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Nova Locação</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/trator (2).png</normaloff>../assets/trator (2).png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Todos (2)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/trator.png</normaloff>../assets/trator.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ativos (1)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/escavadora.png</normaloff>../assets/escavadora.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizado (1)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/trator (1).png</normaloff>../assets/trator (1).png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tabela">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>971</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cliente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Equipamento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Local</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Referencia</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Qtd</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data Inicio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NFe Remessa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NFe Retorno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Observações</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
